android/src/main/java/com/reactnativestripesdk/CardFieldView.kt

```diff
@@ -197,6 +200,19 @@ class CardFieldView(context: ThemedReactContext) : FrameLayout(context) {
     mCardWidget.postalCodeEnabled = isEnabled
   }
 
+  fun setCountryCode(countryCode: String?) {
+    val defaultLocale = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
+      if (context.resources.configuration.locales.isEmpty) "US"
+      else context.resources.configuration.locales[0].country
+    } else {
+      context.resources.configuration.locale.country
+    }
```

Comment:
[`LocaleListCompat#getAdjustedDefault`](https://developer.android.com/reference/kotlin/androidx/core/os/LocaleListCompat#getAdjustedDefault()) gets you the default locale for any version.